### ISSUE TITLE
Machine IP could not be feteched

### ISSUE THREAD

**Issue by kkresek-mentor** [kitematic/kitematic#1095]
I'm working on a Win7 environment.  I downloaded the Docker tool kit that included kitematic.  After I installed Docker tool kit it made my VirtualBox install unusable.  When I would try to open VirtualBox on its own I would get the following error "Failed to create the VirtualBoxClient COM object. The application will now terminate. Callee RC: REGDB_E_CLASSNOTREG (0x80040154)".  I would also get the following error when I started kitematic "Error creating machine: exit status 1 You will want to check the provider to make sure the machine and associated resources were properly removed." 

I tried uninstalling both VirtualBox and Docker tool kit and reinstalling the tool kit.  I got the same errors.  So I uninstalled VirtualBox and manually installed VirtualBox.  This fixed my VirtualBox problem.  I also no longer getting the "Error creating machine: exit status 1 You will want to check the provider to make sure the machine and associated resources were properly removed." from kitematic.  However, I'm now getting "Machine IP could not be fetched. Please retry the setup. If this fails please file a ticket on our GitHub repo."  I tried that "Retry Setup" button but it didn't work.  So I'm logging an issue like it says.

Help would be appreciated. Thanks.

**Comment by FrenchBen** [comment]
Hi @kkresek-mentor - not sure why Virtualbox was having issues, did you previously have VirtualBox installed?
To get a better idea of what's going on, would you mind creating the machine by hand via the PowerShell/GitBash?
```
docker-machine rm default
docker-machine -D create --driver virtualbox --virtualbox-memory "2048" default
```

**Comment by buddhafinger** [comment]
Just posted this on another thread regarding Docker Toolbox - It might help with this problem also.


I was experiencing similar issues:

My setup:

OS X 10.10.5
VirtualBox 5.0.4
Docker Toolbox 1.8.2c
Problems experienced:

Quickstart terminal launches default instance and then prompts for password.
Same D-Bus not installed errors in the log.
Unable to successfully recreate the instance using Kitematic (failed as unable to connect to instance)
Resolution:

Deleted the default docker machine and all associated files.
Deleted 'Host only Networks' in Virtualbox - Preferences - Network - Host Only Network.
Launch Kitematic and let it create the default VM (successfully)
Launch Docker QuickStart terminal and it now works without any problems.
I can now launch docker instances via docker quick start term and use kitematic without any issues.

I got the solution from the link below, I assume it was caused by a conflict with an existing host only network. It's also worth noting the D-Bus error still exists, however things seem to be working fine.
